mindsdb/integrations/handlers/auto_sklearn_handler/auto_sklearn_handler.py

```diff
@@ -0,0 +1,49 @@
+from typing import Optional, Dict
+
+import pickle
+import pandas as pd
+import sklearn
+import autosklearn.regression
+
+from mindsdb.integrations.libs.base import BaseMLEngine
+
+
+class AutoSklearnHandler(BaseMLEngine):
+    name = 'auto_sklearn'
+
+    def create(self, target: str, df: Optional[pd.DataFrame] = None, args: Optional[Dict] = None):
+        y = df.pop(target)
+        X = df.copy()
```

Comment:
This could be the error. Try:

```
X = df.copy()
y = X.pop(target)
```

That way, `X` no longer has the target column on it.

Reply:
We can check the `NULL` accuracy once the models train correctly 👍